lib/Mojolicious/Command/Author/generate/dockerfile.pm

```diff
@@ -0,0 +1,87 @@
+package Mojolicious::Command::Author::generate::dockerfile;
+use Mojo::Base 'Mojolicious::Command';
+
+use Mojo::File qw(path);
+use Mojo::JSON qw(encode_json);
+
+has description => 'Generate "Dockerfile"';
+has usage       => sub { shift->extract_usage };
+
+sub run {
+  my $self = shift;
+  my $name = $self->app->moniker;
+  my $exe  = $ENV{MOJO_EXE} ? path($ENV{MOJO_EXE})->to_rel($self->app->home)->to_string : "script/$name";
+  $self->render_to_rel_file('dockerfile', 'Dockerfile', {name => $name, cmd => encode_json [$exe, 'prefork']});
+}
+
+1;
+
+=encoding utf8
+
+=head1 NAME
+
+Mojolicious::Command::Author::generate::dockerfile - Dockerfile generator command
+
+=head1 SYNOPSIS
+
+  Usage: APPLICATION generate dockerfile [OPTIONS]
+
+    mojo generate dockerfile
+
```

Comment:
Think it might be worth mentioning that generating a Makefile is also required for getting this to work.

Reply:
But this is not really the place. That's what we will have the cookbook recipe for.

Reply:
If it really becomes a problem for people, we will just add a check in the code for a `Makefile.PL` or `cpanfile` and print a warning when generating the `Dockerfile`.

Reply:
I guess I'm fine with having it in the cookbook. It was just the first thing that bit me when I generated a helloworld app for testing the generator. (The second was that I used mojo generate for the dockerfile instead of ./script/myapp generate :)

Reply:
I've adjusted the examples to use `./script/my_app generate dockerfile` consistently, which hopefully is an acceptable first step to help encourage appropriate usage? :innocent: